src/frontend/sharedComponents/MediaScrollView/PhotoThumbnail.tsx

```diff
@@ -43,12 +41,10 @@ export const PhotoThumbnail: FC<PhotoThumbnailProps> = props => {
     <TouchableOpacity
       style={[styles.thumbnailContainer, {width: size, height: size}, style]}
       onPress={onPress}>
-      {isCapturing && !error ? (
-        <ActivityIndicator />
-      ) : uri === undefined ? (
-        <Progress.Circle size={30} indeterminate={true} />
```

Comment:
im not sure why this was here. if the uri is undefined, that should either be reflected in the loading or error state, depending on why the uri is undefined.

Reply:
as to the real cause of the `uri` being undefined, see usages of the `MediaScrollView` component:

```ts
            <MediaScrollView
              photos={attachmentUrls.map(attachmentData => {
                return !attachmentData
                  ? undefined
                  : {
                      thumbnailUri: attachmentData.url,
                      id: attachmentData.driveDiscoveryId,
                    };
              })}
              observationId={observationId}
            />
```